Simplified/Book/UI/NYPLBookDetailViewController.m

```diff
@@ -195,6 +207,32 @@ - (void)didSelectMoreBooksForLane:(NYPLCatalogLane *)lane
   [self.navigationController pushViewController:viewController animated:YES];
 }
 
+- (void)didSelectViewIssuesForBook:(NYPLBook *)book sender:(id)__unused sender
```

Comment:
both these methods were just moved up to group them with other methods of `NYPLBookDetailViewDelegate`.